Model/CustomCustomerAttributeSyncedToYotpoCustomer.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Yotpo\Core\Model;
+
+/**
+ * Placeholder class for SMSBump CustomCustomerAttributeSyncedToYotpoCustomer
+ * Add custom attribute to customer
+ */
+class CustomCustomerAttributeSyncedToYotpoCustomer
```

Comment:
Just to understand: This is a placeholder class for SMSBump CustomCustomerAttributeSyncedToYotpoCustomer, which is actually a data patch.
The ability the override the class from Core by the one that exists in Messaging, is defined through messaging's di.xml `argument`?

Reply:
Yeah, this is just like a dummy class so that there won;t be any execution errors if SMS module is not available in the system for a merchant'e environment.

The real code resides in messaging'ss di.xml and resultant classes are there. This is written in such a way that, if SMS module is installed, it should execute sms module's class. Otherwise it will just be happy with the placeholder class. This is how we are managing the dependency between Core and SMS module in such situations.